content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -1,72 +1,48 @@
-
 This is an article about Sockets. It will give you a glimpse of how clients and servers communicate over the internet.
-
 ### Introduction
 Socket programming is a way of connecting two computers over a network to allow the exchange of information.The connection can be through **connection-oriented protocol** or **connectionless**. In our case, we will be using  TCP/IP a connection-oriented protocol.
-
 In connection-oriented protocols, two computers must establish a connection before transferring data. Whereas in connectionless, you do not need every packet to arrive at its desired destination. This is made possible by UDP(User Datagram Protocol)
-
-We will be using the Client\Server architecture to illustrate more on sockets. Client and server communicate by writing to and reading from socket connection.To establish this connection,the machines must have each other's network information.
-
+We will be using the Client\Server architecture to illustrate more on sockets. Client and server communicate by writing to and reading from the socket connection. To establish this connection, the machines must have each other's network information.
 Table of contents:
 - [Definition-of-a-Socket](#definition-of-a-socket)
 - [Server-Side-Programming ](#server-side-programming)
 - [Client-Side-Programming](#client-side-programming)
 - [How-to-Run-the-Programs-on-different-applications](#how-to-run-the-programs-on-diffrent-applications)
 - [Conclusion](#conclusion)
-
-
 ### Definition of a Socket
-A socket is an **endpoint** of communication, acting as a connection between two computers on a network. It contains a socket number that the TCP layer can use to identify the application to which data will be transmitted. Additionally an endpoint is made up of a port number and an IP address.
-
+A socket is an **endpoint** of communication, acting as a connection between two computers on a network. It contains a socket number that the TCP layer can use to identify the application to which data will be transmitted. Additionally, an endpoint is made up of a port number and an IP address.
 There are two types of sockets
-
 1. Server socket - which waits for a request from clients
 2. Client socket - can be used to send and receive a request from clients.
-
 The client must know two information about the server:
 1.IP address of Server and 
 2.Port number
-
->Port number 0-1024 are reserved for administrative purposes(e,g 21 for FTP,23 for Telnet,25 for email and 80 for HTTP).In our program we will use port number 5000.
-
+>Port number 0-1024 are reserved for administrative purposes(e,g 21 for FTP,23 for Telnet,25 for email and 80 for HTTP).In our program, we will use port number 5000.
 Whenever we create a socket connection in Java, the following steps take place:
```

Comment:
> >Port number 0-1024 are reserved for administrative purposes(e,g 21 for FTP,23 for Telnet,25 for email and 80 for HTTP).In our program, we will use port number 5000.
Whenever we create a socket connection in Java, the following steps take place:

Put a new line before *Whenever we create a socket connection in Java, the following steps take place:*. It was a part of the blockquote when it shouldn't have been so this will fix it. Make sure to check your article in a markdown viewer so you will catch errors like this.